irc/client.go

```diff
@@ -25,17 +25,17 @@ import (
 )
 
 const (
-	// IdleTimeout is how long without traffic before a client's considered idle.
+	// RegisterTimeout is how long clients have to register before we disconnect them
+	RegisterTimeout = time.Minute
+	// IdleTimeout is how long without traffic before a registered client is considered idle.
 	IdleTimeout = time.Minute + time.Second*30
-	// QuitTimeout is how long without traffic (after they're considered idle) that clients are killed.
+	// QuitTimeout is how long without traffic before an idle client is disconnected
 	QuitTimeout = time.Minute
 	// IdentTimeoutSeconds is how many seconds before our ident (username) check times out.
 	IdentTimeoutSeconds = 1.5
```

Comment:
Hmm, we should actually move these constants to `idletimer.go` as well, since they're only used there